olps/strategies/olmar_strategy.py

```diff
@@ -0,0 +1,39 @@
+from .strategy import Strategy
+from ..datasources.olmar_datasource import OLMARDataSource
+import numpy as np
+import cvxpy as cp
```

Comment:
I see cvxpy imported here, but not used anywhere? Is it needed? Just wondering so we can remove the dependency if it's not needed